src/components/intercom/index.tsx

```diff
@@ -84,31 +84,35 @@ export const Intercom: FC<Props> = ({
   autoload = false,
   userInfo = {},
 }) => {
-  const intercomSettings = {
-    cfy: true,
-    app_id: appId,
-    stage,
-    custom_launcher_selector: `#${intercomLauncherId}`,
-    hide_default_launcher: true,
-    alignment: "right",
-    horizontal_padding: 20,
-    vertical_padding: 76,
-    language_override: language,
-    ...userInfo,
-  }
+  const intercomSettings = useMemo(() => {
```

Comment:
Do we really need `useMemo`? We can also remove it because we are using `useDeepCompareEffect`

Reply:
yes I think we need it because `intercomSettings` depends on props. Otherwise the object would be recreated on every render cycle